editors/code/package.json

```diff
@@ -44,7 +44,9 @@
         "vscode": "^1.1.26"
     },
     "activationEvents": [
-        "onLanguage:rust"
+        "onLanguage:rust",
+        "onCommand:rust-analyzer.analyzerStatus",
+        "onCommand:rust-analyzer.collectGarbage"
```

Comment:
Hm, I was under the impression that extension activates on the commands bellow automatically?